src/libraries/Common/src/System/Security/Cryptography/ECCng.HashAlgorithm.cs

```diff
@@ -16,7 +16,7 @@ internal static partial class ECCng
         /// </summary>
         internal static Interop.BCrypt.ECC_CURVE_ALG_ID_ENUM GetHashAlgorithmId(HashAlgorithmName? name)
         {
-            if (name.HasValue == false || string.IsNullOrEmpty(name!.Value.Name))
+            if (name is null || string.IsNullOrEmpty(name.Value.Name))
```

Comment:
Are you sure with this change `HashAlgorithmName` is struct wouldn't be null

Reply:
It is nullable, so it will have equivalent behaviour as before.